[2025-07-20T19:26:35.558+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-20T19:26:35.586+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_weather_forecast.fetch_forecast scheduled__2025-07-20T00:00:00+00:00 [queued]>
[2025-07-20T19:26:35.594+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_weather_forecast.fetch_forecast scheduled__2025-07-20T00:00:00+00:00 [queued]>
[2025-07-20T19:26:35.595+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-20T19:26:35.607+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_forecast> on 2025-07-20 00:00:00+00:00
[2025-07-20T19:26:35.615+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fetch_weather_forecast', 'fetch_forecast', 'scheduled__2025-07-20T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/fetch_weather_data.py', '--cfg-path', '/tmp/tmp0t957rd9']
[2025-07-20T19:26:35.617+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask fetch_forecast
[2025-07-20T19:26:35.617+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=169) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-20T19:26:35.618+0000] {standard_task_runner.py:63} INFO - Started process 245 to run task
[2025-07-20T19:26:35.665+0000] {task_command.py:426} INFO - Running <TaskInstance: fetch_weather_forecast.fetch_forecast scheduled__2025-07-20T00:00:00+00:00 [running]> on host 0189e20d7959
[2025-07-20T19:26:35.753+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_weather_forecast' AIRFLOW_CTX_TASK_ID='fetch_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-07-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-20T00:00:00+00:00'
[2025-07-20T19:26:35.755+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-20T19:26:46.178+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-20T19:26:46.179+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-20T19:26:46.186+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=fetch_weather_forecast, task_id=fetch_forecast, run_id=scheduled__2025-07-20T00:00:00+00:00, execution_date=20250720T000000, start_date=20250720T192635, end_date=20250720T192646
[2025-07-20T19:26:46.221+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-20T19:26:46.234+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-20T19:26:46.236+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
